Options.lua

```diff
@@ -143,6 +143,17 @@ local optionProcessor =
 		end
 		return a_Idx + 2
 	end,
+	
+	-- -sy <pattern> -- Loads all symbols matching the pattern into the sandbox.
+	["-sy"] = function (a_Args, a_Idx, a_Options)
```

Comment:
Ah, that is an unused leftover, command fuzzing is no longer done using the options, but rather in the scenario file. You can remove that option.